pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -1029,6 +1031,13 @@ contract ManagedPool is BaseWeightedPool, ProtocolFeeCache, ReentrancyGuard {
             uint256 fractionalTimePeriod = elapsedTime.divDown(365 days);
             bptAmount = annualizedFee.mulDown(fractionalTimePeriod);
 
+            // Compute the protocol's share of the AUM fee
+            uint256 protocolBptAmount = bptAmount.mulUp(getProtocolAumFeePercentageCache());
+            bptAmount = bptAmount.sub(protocolBptAmount);
+
+            _payProtocolFees(protocolBptAmount);
+
+            // Only show the amount collected by the manager
```

Comment:
Why?

Reply:
Why show only the manager amount? The event is called ManagementAumFeeCollected. Or do you mean something else?